packages/common/src/jsonApiClient/modify/modifyIncludes/stripRelationshipNotToModify.js

```diff
@@ -0,0 +1,28 @@
+function stripRelationshipNotToModify({ relationship } = {}) {
+  const isArray = Array.isArray(relationship.data)
```

Comment:
looks like relationship could be undefined (but don't know if it ever would be), in which case this will throw.

could have the check in line 14 first and add `if (!relationship || !relationship.data)`

Reply:
It could not be the way the function is used but i fix